server/heroku/processes.go

```diff
@@ -100,23 +99,19 @@ func (h *PostProcess) ServeHTTPContext(ctx context.Context, w http.ResponseWrite
 	}
 
 	if form.Attach {
-		inStream, outStream, err := hijackServer(w)
-		if err != nil {
-			return err
+		stream := &hijack.HijackReadWriter{
+			Response: w,
+			Header:   "HTTP/1.1 200 OK\r\nContent-Type: application/vnd.empire.raw-stream\r\n\r\n",
```

Comment:
Super minor, but this might be nice if we just passed an `http.Header` here and did the rest of the formatting within HijackReadWriter.